TakePOS Auto Order improvements

andreubisquerra opened the pull request.

andreubisquerra committed:
TakePOS Auto Order improvements

andreubisquerra committed:
Fix travis

andreubisquerra committed:
Fix URLs

andreubisquerra committed:
Name fix

eldy commented on htdocs/takepos/genimg/qr.php:
```diff
@@ -29,5 +29,8 @@
+
 $module = new modTcpdfbarcode($db);
-$result = $module->buildBarCode("http://www.takepos.com", 'QRCODE', 'Y');
+$result = $module->buildBarCode($urlwithroot."/takepos/auto_order.php?key=".dol_encode($key), 'QRCODE', 'Y');
```
Do you mean $urlwithroot."/takepos/public/auto_order.php    ?

andreubisquerra replied:
You are right @eldy 
I had this fix and many other improvements for the next PULL REQUEST

andreubisquerra committed:
Update qr.php

frederic34 committed:
Update dolistore.class.php

eldy committed:
Trans

eldy committed:
CSS

eldy committed:
CSS

eldy committed:
Fix to solve feedbacks of
https://www.dolibarr.fr/forum/t/email-collector-avec-office-365-bug-et-contournement/32726

eldy committed:
FIX dol_concatdesc to escape <
Better help

eldy committed:
Position of fields

eldy committed:
Merge pull request #13879 from frederic34/patch-4

Update dolistore.class.php

eldy committed:
Merge branch 'develop' into master
